integration-tests/oidc-wiremock/src/main/java/io/quarkus/it/keycloak/PermissionResource.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.it.keycloak;
```

Comment:
@cemnura Please drop this resource and also Vertx resource below(and correspondiing tests), as it is not critical for proving the wiremock approach works, here we can just try Admin and User resources

Reply:
@cemnura you can safely drop this resource :-) as it is not tested anylonger